### PR TITLE
Handle client side error

### PR DESCRIPTION
**Pull request by d8660091**
**Description of changes**
Fixed some warnings btw

**Type of Change**

- [ ] Bug
- [ ] Styling
- [x] New Feature

**Checklist before requesting review:**

- [x] Matches design:

  - component sizes, spacing, and styles
  - font size, weight, colour
  - spelling has been double checked

- [ ] New uikit components have Storybook stories
- [ ] Feature is minimally responsive.
- [x] Manual testing of UI feature.
- [ ] Selenium test is completed and passing.

### CODE REVIEW

File: components/pages/ClientError.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+          >
+            We are working on fixing the problem and appreciate your patience.{' '}
+            <A href="#" onClick={handleReloadClick}>
+              Reload this page
+            </A>
+            .
+          </Typography>
+        </div>
+        <div>
+          <img alt="Bug eats wire" src="/static/client-error.svg" />
```
Comment (joneubank) [inline components/pages/ClientError.js]: alt should probably be related to what is being conveyed not the actual image. maybe: `Bug in the code`

File: components/pages/error.js [hunk 2]
```diff
@@ -36,6 +36,8 @@ export default function ErrorLayout({ children }) {
           { displayName: 'Contact', href: '#' },
           { displayName: 'Documentation', href: '#' },
           { displayName: 'Privacy Policy', href: '/privacy' },
+          { displayName: 'Terms & Conditions', href: '#' },
```
Comment (joneubank) [inline components/pages/error.js]: Should this be in this PR?

File: components/pages/submission-system/programs/ProgramsTable.js [hunk 3]
```diff
@@ -97,6 +97,7 @@ export default (tableProps: {
       Cell: ({ original }) => {
         return get(original, 'administrators', []).map((admin, idx) => (
           <A
+            key={admin.email}
```
Comment (joneubank) [inline components/pages/submission-system/programs/ProgramsTable.js]: Is this related to this PR?

File: pages/_error.js [hunk 4]
```diff
@@ -27,11 +28,10 @@ class Error extends React.Component {
     if (errorCode === 401) {
       return <Page401 />;
     }
-    return (
-      <p>
-        {errorCode ? `An error ${errorCode} occurred on server` : 'An error occurred on client'}
-      </p>
-    );
+    if (errorCode) {
+      return <p>An error {errorCode} occurred on server</p>;
```
Comment (joneubank) [inline pages/_error.js]: We don't ever plan on showing a blank white page with a generic error... can we return the 500 page here and console log the specific error?

File: components/pages/submission-system/programs/ProgramsTable.js (hunk 3 above)
Comment (d8660091) [reply components/pages/submission-system/programs/ProgramsTable.js]: It's not related. But it's too trivial to make a ticket for that.

File: components/pages/error.js (hunk 2 above)
Comment (d8660091) [reply components/pages/error.js]: It should be there because the client error page should present the Terms & Conditions link in the footer
